scripts/cmake/Modules/PackagingFedora.cmake

```diff
@@ -0,0 +1,199 @@
+set (CPACK_RPM_PACKAGE_VERSION "${PROJECT_VERSION}")
```

Comment:
Give a short info what this file is about (that it sets CPACK_RPM_ vars and also installs some rpm specific files). Maybe also split this into two files (with separate headers)